tests/bsim/bluetooth/audio/test_scripts/audio_config_tests/_gmap_ac_8_ii.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+dir_path=$(dirname "$0")
+
+set -e # Exit on error
+
+# Run all gmap_ac_8_ii* tests
+for file in "$dir_path"/gmap_ac_8_ii*.sh; do
```

Comment:
Why do you want all these instead of just calling run_parallel directly as a user?
`SEARCH_PATH=tests/bsim/bluetooth/audio/test_scripts/audio_config_tests/gmap_ac_8_ii*.sh  tests/bsim/run_parallel.sh`

Reply:
Running these tests in parallel makes it a lot harder to follow the logs for debugging purposes. In my experience, in most cases, you'll want to stop execution when a test fails (thus the `set -e` above) so that you can spend time determining why that specific test failed.

Reply:
> Running these tests in parallel makes it a lot harder to follow the logs for debugging purposes. 

It should not(?). run_parallel should serialize the logs and only print them for those which failed.